openupgrade_scripts/scripts/rating/14.0.1.0/upgrade_analysis.txt

```diff
@@ -4,6 +4,7 @@ rating       / rating.rating            / is_internal (boolean)         : NEW is
 ---XML records in module 'rating'---
 NEW ir.actions.act_window: rating.rating_rating_view
 DEL ir.actions.act_window: rating.action_view_rating
+ir.model.constraint: rating.constraint_rating_rating_rating_range (changed definition: is now 'check(rating >= 0 and rating <= 5)' ('check(rating >= 0 and rating <= 10)'))
```

Comment:
This will require adaptation and maybe something we have missed on the migration of such field?

Reply:
We haven't missed, already done in pre-migration :)